Makefile

```diff
@@ -105,6 +105,16 @@ test: generate fmt vet manifests
 test-e2e: generate fmt vet
 	GOFLAGS=$(GOFLAGS) USE_EXISTING_CLUSTER=true go test -v ./test -coverprofile cover.out -race -args -ginkgo.v -ginkgo.trace
 
+.PHONY: kind-deploy-olm
+kind-deploy-olm:
+	kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.17.0/crds.yaml
+	sleep 5
+	kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.17.0/olm.yaml
+
+.PHONY: set-olm-e2e-img
+set-olm-e2e-img:
```

Comment:
I would rename this to something like `deploy-with-olm` and we should use `kustomize` to configure the image and possibly the namespace, then `kustomize build` and `kubectl apply` similar to the `deploy` make recipe.